core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -327,7 +329,8 @@ namespace kagome::blockchain {
       }
     }
 
-    metric_known_chain_leaves_->set(tree_->getMetadata().leaves.size());
+    metric_known_chain_leaves_->set(
+        static_cast<double>(tree_->getMetadata().leaves.size()));
```

Comment:
Why is the number of known leaves a double?

Reply:
Honestly, it's a linter silencer. And yes, metrics::Gauge has: 
virtual void set(double val) = 0;